config/settings.yml.example

```diff
@@ -22,6 +22,8 @@
 # (default: true)
 #:forward_verify: true
 
+# if you run foreman on sub-uri (e.g. http://127.0.0.1:3000/foreman/) you must
+# specify the trailing slash
```

Comment:
This works, but I think it would be less error prone if we automatically added the missing trailing slash when it's missing...